src/aspire/image/image.py

```diff
@@ -606,13 +610,13 @@ def save(self, mrcs_filepath, overwrite=None):
 
         with mrcfile.new(mrcs_filepath, overwrite=overwrite) as mrc:
             # original input format (the image index first)
-            mrc.set_data(self._data.astype(np.float32))
+            mrc.set_data(data)
             # Note assigning voxel_size must come after `set_data`
             if self.pixel_size is not None:
                 mrc.voxel_size = self.pixel_size
 
     @staticmethod
-    def load(filepath, dtype=None):
+    def _load(filepath, dtype=None):
```

Comment:
To make things more explicit, could we call this `_load_raw`?

Reply:
Sounds good to me. Done.